setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = skt
+description = "Sonic Kernel Testing"
+long_description = file: README.md
+version = 0.0.1
+author = Red Hat, Inc.
+license = GPL
+
+[options]
+packages = find:
+include_package_data = True
+
+[options.entry_points]
+console_scripts =
+    skt = skt.executable:main
```

Comment:
Maybe the path `bin/skt` is a more intuitive place to put it and if in the future we have more than one executable we already have a canonical folder to put it.

Reply:
This should cause the `skt` executable to land in `~/.local/bin` or `/usr/bin` on the installer's system. We could add extra executables by adding more `console_scripts` entries later if we need to.

Reply:
You're right, `console_scripts` put the binary in the right place when you install the project. But I was talking about inside the repository not in the delivery stage.

Reply:
Oh, I see what you're saying now. Let me see what I can do.

Reply:
@agustinhenze To be fair, you could use `pip install -e .` in the skt directory to install the package in develop mode. You'll be able to make changes within the local repository without needing to fully reinstall the package. Would that be sufficient?

Reply:
@major I agree with you that using `pip install -e .` is the recommended way to development. Maybe we should mention it on the README.

I'm not explaining properly, sorry. The name that you picked `executable.py` is not the happiest choice for me. What I'm proposing you is, you move the skt.py to bin/skt. I think that's a more intuitive path for the file and also fixes the issue #9.